UG - Include labelled images for set functions (#169)
AY2021S1-CS2113T-W13-3/tp: docs/UserGuide.md

![Summary](diagrams/Summary.png)

> 
> 

Reviewer: ```suggestion
```

When you have typed in a command that is inconsistent with what is required as per the tutorial's
instruction for that particular step, an invalid command message will be returned until you type 

> in the correct command in the terminal line as shown below. r

Reviewer: ??
```suggestion
in the correct command in the terminal line as shown below.
```

    * `v` represents a voltage source
* The `VALUE` can be an integer of a float, of which the units depends on the component set.

> The following diagram is an example of when the `set r 500` command is used in any circuit template:

Reviewer: For consistency.
```suggestion
The image below shows an example of using the command `set r 500` on the template shown.
```

`set r 500`

> Expected Outcome:

Reviewer 2: Perhaps instead of having example of usage + expected output, could you combine it into 1 sentence that reads something like: "the following diagram is an example of when `set r 500` command is used in a valid template"

Author: Okay, changed.

> ![SetCircuit](diagrams/SetCircuitLabel.png)

Reviewer 2: For this image could you change the green arrow to point at "the resistor was set to 500" instead of total resistance:
![image](https://user-images.githubusercontent.com/57522870/98458261-c740fc00-21c9-11eb-9fb9-0f8b6bdff116.png)

Author: Okay, changed!

The image below shows an example of using the command `add parallel r 500` on a template.

> ![AddCircuit](diagrams/SetCircuitLabel.png)

Reviewer: Why is this changed?
```suggestion
![AddCircuit](diagrams/AddCircuitLabel.png)
```

> Expected outcome:

Reviewer 2: Same for this

Author: Okay, changed.

* The `INPUT` can be chosen only from the current `template`.
* The `VALUE` can be any *integer*, but any valid non-zero *integer* will be treated as `1`, while 0 is treated as `0`. Valid *integers* are defined as the *Java* `int` type.

> The following diagram is an example of when the `set B 0` command is used in any boolean template:

Reviewer: ```suggestion
The image below shows an example of using the command `set B 0` on the template shown.
```

> ![AddBoolean](diagrams/SetBooleanLabel.png)

Reviewer: ```suggestion
![SetBoolean](diagrams/SetBooleanLabel.png)
```
